Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/chapter2_material_widgets/Tutorial2_4Image.kt

```diff
@@ -128,7 +129,10 @@ private fun TutorialContent() {
                         "set it to Image component."
             )
 
-            ImageDownloadWithGlideExample()
+            // Previews rendered within Android Studio cannot access Network to load images. Skipping this Composable while rendering Preview.
+            if (!isInPreview) {
+                ImageDownloadWithGlideExample()
```

Comment:
Previews rendered within Android Studio cannot access Network to load images. Hence Preview crashes if we try to load an image in a Composable.

There are 2 ways we can approach this problem.
1. Show a static image in `@Preview` instead of loading one from the network.
2. Skip a certain composable (that is trying to access network) for Previews only. This composable will be part of live app.

For #2, we can use `LocalInspectionMode.current` to check a Composable is in Preview. `LocalInspectionMode.current` returns `true` if the composition is composed inside an Inspectable component.